src/defaults/html.pipe.js

```diff
@@ -44,6 +44,10 @@ function hascontent({ content }) {
   return !(content !== undefined && content.body !== undefined);
 }
 
+function paranoid() {
+  return !!process.env.HLX_SANITIZE_DOM;
```

Comment:
We should pass the `paranoid` flag through params, not through environment vars, as the latter can't be set easily on a function-by-function basis.

Reply:
it could be set in the pre.js :-)
```
if (action.params.paranoid) {
  process.env.HLX_SANITIZE_DOM = true;
}
```

but I'll add it to the `secrets`

Reply:
I think each invocation is running in a separate process, but it's not a good idea anyway.